test/smoke/rhel-docker/perm_test.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Cookbook:: nexus_repository_manager
+#
+# Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
+#
+
+# Permissions test for recipe nexus_repository_manager::rhel-docker
+
+control 'rhel-docker-perms-test-001' do
+  title 'Allows nexus user to mutate /etc/password'
+  describe user('nexus') do
+    it { should exist }
+  end
+
+  describe file('/etc/passwd') do
+    it { should be_allowed('write', by_user: 'nexus') }
```

Comment:
This test is failing. I am not sure why yet.

Reply:
Test is now passing after upgrading the ami used for chef testing with the latest chef version on jenkins.